pkg/chain_iterator/block_batch_iterator.go

```diff
@@ -52,7 +52,6 @@ type BlockBatchIterator struct {
 	current            *types.Header
 	onBlocks           OnBlocksFunc
 	isEnd              bool
-	reverse            bool
```

Comment:
Its still in `BlockBatchIteratorConfig`?

Reply:
Deleted.